iterfzf/__init__.py

```diff
@@ -30,10 +30,11 @@ def iterfzf(
     # Search mode:
     extended: bool = True,
     exact: bool = False,
-    case_sensitive: bool = None,
+    case_sensitive: Optional[bool] = None,
     # Interface:
     multi: bool = False,
     mouse: bool = True,
+    bind: Optional[Dict[str, str]] = None,
```

Comment:
IMHO it's better to be abstract [`Mapping`][1] instead of concrete `Dict` here.

[1]: https://docs.python.org/3.8/library/typing.html#typing.Mapping

Reply:
I agree, abstract is better.